packages/audioplayers/lib/src/audioplayer.dart

```diff
@@ -88,9 +97,6 @@ class AudioPlayer {
 
   ReleaseMode get releaseMode => _releaseMode;
 
-  /// Creates a new instance and assigns an unique id to it.
-  AudioPlayer({String? playerId}) : playerId = playerId ?? _uuid.v4();
-
```

Comment:
Readd constructor here:

```suggestion
  /// Creates a new instance and assigns an unique id to it.
  AudioPlayer({String? playerId}) : playerId = playerId ?? _uuid.v4() {
    _onPlayerCompleteStreamSubscription = onPlayerComplete.listen((_) {
      state = PlayerState.completed;
    });
  }

```

Reply:
Sorry, I somehow missed this. I see you are adding commits, so should you add this change too?

Reply:
I did all the changes. Hope you agree with them. If not let me know ;D